frontend/src/bundles/chat/components/chat/chat.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box } from '~/bundles/common/components/components.js';
+
+import {
+    type GenerateTextRequestDto,
+    type Message,
+} from '../../types/types.js';
+import { ChatBody } from '../chat-body/chat-body.js';
+import { ChatFooter } from '../chat-footer/chat-footer.js';
+import { ChatHeader } from '../chat-header/chat-header.js';
+
+type Properties = {
+    messages: Message[];
+    onSendMessage: (payload: GenerateTextRequestDto) => void;
+    headerTitle: string;
+    headerComment: string;
+};
+
+const Chat: React.FC<Properties> = ({
+    messages,
+    onSendMessage,
+    headerTitle,
+    headerComment,
+}) => {
+    return (
+        <Box>
+            <ChatHeader title={headerTitle} comment={headerComment} />
+            <Box sx={{ p: '10' }}>
```

Comment:
Here you can add maxHeight:  and overflowY: 'auto' so the ChatBody doesnt grow too much when there are a of of messages

Reply:
@stefano-lacorazza Thanks, already is has a maxHeight: and overflowY: 'auto' in ChatBody component
![image](https://github.com/user-attachments/assets/988cba54-81a2-4a25-b3bf-acf47b828c68)